index.html

```diff
@@ -3,59 +3,74 @@
 <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>[APPNAME]</title>
-
-    <!-- stylesheet -->
+    <title>Meal Plan App</title>
+    <link href="https://fonts.googleapis.com/css2?family=Inter:wght@400;500;600;900&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="styles/styles.css">
-
-    <!-- Google Fonts -->
-
 </head>
 <body>
-    <main>
-        <form action="">
-            <!-- welcome section -->
-            <fieldset>
-                <h1>Hi, Welcome to [APPNAME]</h1>
-                <p>[APPNAME] pairs you the with perfect meal plan based on your dietary preference, restrictions and caloric intake</p>
-                <input type="button" value="Get Started!">
+    <main class="quiz-section">
+        <form action="#" method="#" name="diet-quiz" class="quiz-section__form" id="">
+            <fieldset class="form__stage form__stage--center-align">
```

Comment:
There is both `_` and `-` used here. If we're using kabob case I think we should stick to one for style points. Also, what is the `--` and `__`? I have never seen that before.

Reply:
I don't mind combing through to make the changes 🍢